lms/djangoapps/commerce/api/v0/views.py

```diff
@@ -113,6 +114,17 @@ def post(self, request, *args, **kwargs):
         honor_mode = CourseMode.mode_for_course(course_key, CourseMode.HONOR)
         audit_mode = CourseMode.mode_for_course(course_key, CourseMode.AUDIT)
 
+        # Check to see if the User has an entitlement and enroll them if they have one for this course
+        if CourseEntitlement.check_for_existing_entitlement_and_enroll(
```

Comment:
We are calling .check_for_existing_entitlement_and_enroll in two places? are there two seperate cases/flows for this during login?

Reply:
Yes. One is from the Marketing site the other is from the LMS Course About pages that are used by WL. They come in from different APIs...  Needs to be fixed in the future.

Reply:
Got it. Good spot on putting this in both places, should we give a heads up to white-label about this kind of thing? They are the ones that use LMS based course about pages right?